.travis.yml

```diff
@@ -2,9 +2,6 @@ language: python
 python:
 - 2.7
 - 3.5
-
-sudo: "required"
```

Comment:
I thought "sudo" has to be turned of to set its value to false. I cannot find anything about "required". Mind explaining why we need sudo?